cdap-app-fabric/src/main/java/co/cask/cdap/app/queue/QueueSpecificationGenerator.java

```diff
@@ -50,21 +52,31 @@ public static Node stream(String name) {
     }
 
     public Node(FlowletConnection.Type type, String name) {
+      this(type, null, name);
+    }
+
+    public Node(FlowletConnection.Type type, @Nullable String namespace, String name) {
       this.type = type;
+      this.namespace = namespace;
       this.name = name;
     }
 
+
     public FlowletConnection.Type getType() {
```

Comment:
Public interface with public method with no docs!
Can you please put java docs in all these methods.